metacat-main/src/main/java/com/netflix/metacat/main/configs/SNSNotificationsConfig.java

```diff
@@ -52,18 +46,14 @@
     /**
      * If SNS notifications are desired and no existing client has been created elsewhere
      * in the application create a default client here.
-     * @param config       The system configuration abstraction to use
-     * @param registry     registry for spectator
+     *
      * @return The configured SNS client
      */
     //TODO: See what spring-cloud-aws would provide automatically...
```

Comment:
Still think this TODO is worthwhile. In Genie 4 we're using this.

Reply:
Can u point to the code?

Reply:
There's not much code since it's a lot of auto configs and whatnot but... https://github.com/Netflix/genie/blob/master/genie-agent/src/main/java/com/netflix/genie/agent/configs/AwsConfig.java

https://github.com/Netflix/genie/blob/master/genie-agent/src/main/java/com/netflix/genie/GenieAgentApplication.java

Still needs cleanup and testing. Reading docs probably would be more of a help.

Reply:
Looking at the documentation, I see that it is an umbrella lib for clients to all(RDS, S3, SNS, SQS). I'll keep the TODO :) and make the change later.